SummerProject/static/src/components/Registration/index.js

```diff
@@ -1,53 +1,144 @@
+                                </div>
+                            </div>
+                            <div className="form-group ">
+                                <div className={validation.lastName.isInvalid && 'has-error'}>
+                                    <label>Last Name</label>
+                                    <input type="text" placeholder="Doe" name="lastName" value={lastName}
+                                           onChange={this.changeHandler} className="form-control"/>
+                                    <span className="help-block">{validation.lastName.message}</span>
+                                </div>
+                            </div>
                         </div>
-                        <div className="form-group ">
-                            <label>Last Name</label>
-                            <input type="text" placeholder="Doe" name="last_name" className="form-control"/>
+                        <div className="passw-email">
+                            <div className={validation.email.isInvalid && 'has-error'}>
+                                <label>Email</label>
+                                <div className="form-group">
+                                    <input type="text" placeholder="johndoe@gmail.com" name="email" value={email}
+                                           onChange={this.changeHandler}
+                                           className="form-control input-height"/>
+                                    <span className="help-block">{validation.email.message}</span>
+                                </div>
+                            </div>
+                            <div className={validation.password.isInvalid && 'has-error'}>
+                                <label>Password</label>
+                                <div>
+                                    <div className="form-group">
+                                        <input type="password" placeholder="4 + caharacters" name="password"
+                                               value={password} onChange={this.changeHandler}
+                                               className="form-control input-height"/>
+                                        <span className="help-block">{validation.password.message}</span>
+                                    </div>
+                                </div>
+                            </div>
                         </div>
-                    </div>
-                    <div className="passw-email">
-                        <label>Email</label>
-                        <div className="form-group">
-                            <input type="text" placeholder="johndoe@gmail.com" name="email"
-                                   className="form-control input-height"/>
+                        <div className="sign-up-w">
+                            <button type="submit" onClick={this.submitHandler} className="btn-primary sing-up">SIGN UP
```

Comment:
You have two triggers for `submitHandler` (one here and one on line `88`). You only need one, so please remove one of them

Reply:
Done